src/main/java/presenters/OutputPresenter.java

```diff
@@ -0,0 +1,56 @@
+package presenters;
+
+/**
+ * This presenter class contain methods that output game progress information
+ * in the Game Progress Panel.
+ * NOTE: Since we are at the terminal version, this class is minimally implemented.
+ */
+public class OutputPresenter {
+
+    /**
+     * Print a message at the terminal
+     * which announces that a player has invested / upgraded a property.
+     */
+    public static void notifyOwnerUpgraded(int playerId, String verb,
+                                           String propName, int currPrice, int level) {
+        System.out.println("Player " + playerId + " " + verb + " " + propName +
+                " for " + currPrice + " TBucks.");
+        System.out.println(propName + " is now at level " + level + ".");
+    }
+
+    /**
+     * Print a message at the terminal
+     * which announces that a player has ignored a property.
+     */
+    public static void notifyOwnerIgnored(int playerId, String propName) {
+        System.out.println("Player " + playerId + " ignored " + propName + ".");
+    }
+
+    /**
+     * Print a message at the terminal
+     * which announces that a player has been charged at a property owned by others.
+     */
+    public static void notifyPasserbyPaid(int passerId, int tax,
+                                          int ownerId, String propName) {
+        System.out.println("PLayer " + passerId + " paid " + tax + " to Player " +
```

Comment:
To be very honest, I don't really like this way of building the string, as if we want to change the format, it could take really long. 

Something like:
`String.format("Player %d paid %d to Player %d at %s.", passerId, tax, ownerId, propName);`

Reply:
Same advice goes to all other strings that are built using the regular concatenation way.

Reply:
Yeah, String.format looks neat. I'll change those first.